### ISSUE TITLE
wrong browsers used with certain browsers+devices configurations

### ISSUE THREAD

**Issue by ValeryVS** [playwright-community/jest-playwright#464]
**Describe the bug**

Wrong browsers used with certain configurations. For all custom devices chromium is used. If devices described with same name and different defaultBrowserType, then it will be runned multiple times, and all times with chromium.

**To Reproduce**

Add some custom devices with defaultBrowserType to config.

**Expected behavior**

With `browsers: ['chromium', 'firefox', 'webkit'],` and `devices: ['iPad Pro 11 landscape', 'iPhone SE', ...some custom configs...]` test should be also runned on those described devices.

**Desktop (please complete the following information):**
 - OS: macOS
 - Playwright 1.6.1
 - jest-playwright 1.4.0

**Jest configuration (Either in the package.json > jest or in the `jest.config.js`):**

 ```js
module.exports = {
  browsers: ['chromium', 'firefox', 'webkit'],
  devices: [
    'iPad Pro 11 landscape',
    'iPhone SE',
    {
      name: 'FHD desktop',
      viewport: {
        width: 1920,
        height: 1080,
      },
      deviceScaleFactor: 1,
      isMobile: false,
      hasTouch: false,
      defaultBrowserType: 'chromium',
    },
    {
      name: 'FHD desktop',
      viewport: {
        width: 1920,
        height: 1080,
      },
      deviceScaleFactor: 1,
      isMobile: false,
      hasTouch: false,
      defaultBrowserType: 'firefox',
    },
  ],
  useDefaultBrowserType: true,
  launchOptions: {
    // headless: false,
  },
};
```

**Comment by mmarkelov** [comment]
@ValeryVS thanks for report. I'll take a look

**Comment by mmarkelov** [comment]
@ValeryVS could you check it in the latest release?

**Comment by ValeryVS** [comment]
@mmarkelov 
Checked. Fixed in 1.4.1, thanks!
